drivers/vtencent/util.go

```diff
@@ -0,0 +1,392 @@
+package vtencent
+
+import (
+	"bytes"
+	"context"
+	"crypto/md5"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"path"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/alist-org/alist/v3/drivers/base"
+	"github.com/alist-org/alist/v3/internal/driver"
+	"github.com/alist-org/alist/v3/internal/model"
+	"github.com/alist-org/alist/v3/pkg/utils"
+	"github.com/go-resty/resty/v2"
+	"github.com/google/uuid"
+)
+
+func (d *Vtencent) request(url, method string, callback base.ReqCallback, resp interface{}) ([]byte, error) {
+	req := base.RestyClient.R()
+	req.SetHeaders(map[string]string{
+		"cookie":       d.Cookie,
+		"content-type": "application/json",
+		"origin":       d.conf.origin,
+		"referer":      d.conf.referer,
+	})
```

Comment:
```suggestion
	}).ForceContentType("application/json")
```
响应头ContentType有问题，所以后面json不解析。可以加上这个解决